hack/verify-api-groups.sh

```diff
@@ -111,12 +111,10 @@ for expected_install_package in "${expected_install_packages[@]}"; do
 		continue
 	fi
 
-	for known_version_file in "${known_version_files[@]}"; do
-		if ! grep -q "${expected_install_package}/install" ${known_version_files} ; then
```

Comment:
```
In ./hack/verify-api-groups.sh line 115:
                if ! grep -q "${expected_install_package}/install" ${known_version_files} ; then
                                                                   ^--------------------^ SC2128: Expanding an array without an index only gives the first element.
                                                                   ^--------------------^ SC2086: Double quote to prevent globbing and word splitting.
```